reference/import_export/connectors.rst

```diff
@@ -0,0 +1,21 @@
+Akeneo Native Connector
+=================
+
+CSV Format
+-------------
+
+Akeneo Native Connector supports reading and writing CSV files. For more information, please refer to the chapters on product import (cf :doc:`/reference/import_export/product-export`) and product export (cf :doc:`/reference/import_export/product-export`) in order to understand how to customize the connector.
```

Comment:
You should put the end of your sentence at its beginning: "In order to customize the connector, you can refer to the chapters about product import and export."